README.md

```diff
@@ -43,40 +43,33 @@ The API recognizes the following properties under the top-level `api` key in you
 |`indexName`|*no*|*pelias*|name of the Elasticsearch index to be used when building queries|
 |`relativeScores`|*no*|true|if set to true, confidence scores will be normalized, realistically at this point setting this to false is not tested or desirable
 |`accessLog`|*no*||name of the format to use for access logs; may be any one of the [predefined values](https://github.com/expressjs/morgan#predefined-formats) in the `morgan` package. Defaults to `"common"`; if set to `false`, or an otherwise falsy value, disables access-logging entirely.|
-|`services`|*no*||service definitions for [point-in-polygon](https://github.com/pelias/pip-service), and [placeholder](https://github.com/pelias/placeholder), and [interpolation](https://github.com/pelias/interpolation) services.  If missing (which is not recommended), the services will not be called.|
+|`services`|*no*||service definitions for [point-in-polygon](https://github.com/pelias/pip-service), [libpostal](https://github.com/whosonfirst/go-whosonfirst-libpostal), [placeholder](https://github.com/pelias/placeholder), and [interpolation](https://github.com/pelias/interpolation) services.  If missing (which is not recommended), the services will not be called.|
 |`defaultParameters.focus.point.lon` <br> `defaultParameters.focus.point.lat`|no | |default coordinates for focus point
 
-Example configuration file would look something like this:
+A good starting configuration file includes this section (fill in the service and Elasticsearch hosts as needed):
 
 ```
 {
   "esclient": {
-    "keepAlive": true,
-    "requestTimeout": "1200000",
-    "hosts": [
-      {
-        "protocol": "http",
-        "host": "somesemachine.elb.amazonaws.com",
-        "port": 9200
-      }
-    ]
+    "hosts": [{
+      "host": "elasticsearch"
+    }]
   },
   "api": {
-    "host": "localhost:3100/v1/",
-    "indexName": "foobar",  
-    "relativeScores": true,
     "services": {
-      "pip": {
-        "url": "http://mypipservice.com:3000"
-      },
       "placeholder": {
-        "url": "http://myplaceholderservice.com:5000"
+        "url": "http://placeholder:4100"
+      },
+      "libpostal": {
+        "url": "http://libpostal:8080"
```

Comment:
I still think we should change this to be in the 4000 range, in order to be consistent with the other services :)

Reply:
We can do that in the dockerfiles repo, but as the [default port for go-whosonfirst-libpostal is 8080](https://github.com/whosonfirst/go-whosonfirst-libpostal#wof-libpostal-server), we can't enforce it everywhere.